lib/sort_by_length.rb

```diff
@@ -1,7 +1,14 @@
 # A method which will return an array of the words in the string
 #  sorted by the length of the word.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n) where n is the length of my_sentence?
+# Space complexity: O(n) where n is the the length of my_sentence -- builds aux. array based on n size?
 def sort_by_length(my_sentence)
```

Comment:
Clever way to implement selection sort, however since you are deleting and finding the smallest each iteration of the `my_sentence.length.times` loop, this is O(n^2) in time complexity.